ui_framework/components/icon/delete_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { KuiIcon } from './kui_icon';
+
+export function DeleteIcon() {
+  return <KuiIcon className="fa-trash"/>;
```

Comment:
> Also, I wonder if we should use CamelCasing for the component file names (will clearly indicate that a file contains a component). In this example, DeleteIcon.js, CreateIcon.js, KuiIcon.js, etc...

I **much** prefer CamelCasing file names, but I think there may have been a reason we use underscore?  I was chatting with... I think @cjcenizal about this.  Case sensitivity on different OS's or something?  Anyway I am totally in favor of that, if there are no relevant objections.  Just want to pull the right people in who have more background then me.  @spalger?

++ @kjbekkelund on the exports suggestion.